android/src/main/java/vn/luongvo/kmm/survey/android/ui/theme/Color.kt

```diff
@@ -1,9 +1,11 @@
 package vn.luongvo.kmm.survey.android.ui.theme
 
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.graphics.Color.Companion.Black
 import androidx.compose.ui.graphics.Color.Companion.White
 
 internal val Black20 = Black.copy(alpha = 0.2f)
+internal val BlackRussian = Color(0xFF15151A)
```

Comment:
Where did you get this naming from, don't see. it in Figma 🤔 Maybe a less ambiguous name `BlackAction`?

Reply:
@blyscuit Android and Flutter teams often try to name a hex color value to a name. We use this tool https://www.htmlcsscolor.com/hex/15151A to get its name 😄

Reply:
The idea: it's a color palette so we should not give it a feature or action name, custom app colors will be at [AppColors](https://github.com/luongvo/kmm-survey/blob/develop/android/src/main/java/vn/luongvo/kmm/survey/android/ui/theme/AppColors.kt) class.

Reply:
👍